src/components/questions-accordion/QuestionsAccordion.tsx

```diff
@@ -0,0 +1,93 @@
+import { FC, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import Accordion from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+
+import ArrowForwardIosSharpIcon from '@mui/icons-material/ArrowForwardIosSharp'
+
+import { AccordionItem } from '~/types'
+import { styles } from '~/components/questions-accordion/QuestionsAccordion.styles'
+
+interface QuestionsAccordion {
+  items: AccordionItem[]
+  title?: string
+  showIcon?: boolean
+}
+
+const QuestionsAccordion: FC<QuestionsAccordion> = ({
+  items,
+  title = '',
+  showIcon = true
+}) => {
+  const [activeQuestions, setActiveQuestions] = useState<number[]>([])
+
+  const onChange = (activeQuestion: number) => {
+    setActiveQuestions((prevActiveQuestions) => {
+      if (prevActiveQuestions.includes(activeQuestion)) {
+        return prevActiveQuestions.filter(
+          (prevActiveQuestion) => prevActiveQuestion !== activeQuestion
+        )
+      } else {
+        return [...prevActiveQuestions, activeQuestion]
+      }
+    })
+  }
+
+  const { t } = useTranslation()
+
+  return (
+    <Box sx={title ? styles.container : {}}>
+      {title ? (
+        <Typography sx={styles.title} variant='h5'>

```

Comment:
We use TS, add to VariantEnum h5 and use it.
variant={VariantEnum .h5}

Reply:
yeap, i know it but i don't see such variant in enums

Reply:
You can add this variant

Reply:
okey 
firstly, I wanted to add to this enums but i thought that this enums are only for variants similar to existing but not for text